packages/EasySchedule/src/Bridge/Symfony/Resources/config/doctrine.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use EonX\EasySchedule\Bridge\Doctrine\CommandExecutedEventListener;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
```

Comment:
Let's rename to `clear_entity_manager_on_command.php`

Reply:
It should be either `...in_command` or `...on_command_execution`.

Reply:
Renamed

Reply:
Thanks